source/django_package/domain/import_datas/csv_config_file.yaml

```diff
@@ -0,0 +1,3 @@
+name: test
+source: https://raw.githubusercontent.com/plotly/datasets/master/2011_february_aa_flight_paths.csv
+format: csv
```

Comment:
Looks like a test - should go into test directory somewhere else.
By the way, you have to write tests for your code.